src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.FormatInfo.cs

```diff
@@ -297,8 +318,11 @@ private static DriveType GetDriveType(string fileSystemName)
 
                 case "9p":
                 case "acfs":
+                case "afp":
```

Comment:
What is returned for the new Mojave default, APFS? Is it AFPFS?

Reply:
I don't have it installed. Maybe @maryamariyan could comment?

Reply:
@ajaybhargavb also may have Mojave handy

Reply:
`GetDriveType(string fileSystemName)` returns `DriveType.Fixed` on Mojave is `qnx4` and it falls under the `qnx4` case.

https://github.com/dotnet/corefx/blob/d96d7a1a98b0623e0171efe3731348942b484149/src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.FormatInfo.cs#L287

Reply:
Thanks. Glad it's covered.